python/ray/util/placement_group.py

```diff
@@ -211,6 +211,24 @@ def remove_placement_group(placement_group: PlacementGroup):
     worker.core_worker.remove_placement_group(placement_group.id)
 
 
+def get_placement_group(placement_group_name: str):
+    """Get a placement group object with a global name.
+
+    Returns:
```

Comment:
Can we just raise a ValueError? (same behavior as `get_actor`). (or lmk if you have a particular reason why it should return None)

Reply:
For user, the most frequency scenarios is:  get a placement group according to a name and then judge whether it exist or not, if not exist, create a new one with the name.
If we raise a ValueError here, user need to catch the exception and create the placement group in `try.. catch..`, I think it's a bad UE.

Reply:
@clay4444 we should keep this consistent with `ray.get_actor`, which currently raises a `ValueError` in this scenario. For now let's follow the same API here -- if we want to return None instead, that's fine, but we should change both of them.

Reply:
ok, I will make them keep same scenario in this pr